locale/en/foundation/case-studies.md

```diff
@@ -16,7 +16,7 @@ preferences (current and future) for front end, back end and IoT developers.
 
 * [Read the full announcement.](/en/blog/announcements/nodejs-foundation-survey/)
 * [See the infographic.](/static/documents/2016-survey-infographic.png)
-* [Read the full report.](/static/documents/2016-survey-report.pdf)
+* <a href="/static/documents/2016-survey-report.pdf" onClick="ga('send', 'event', { eventCategory: 'casestudy', eventAction: 'click', eventLabel: 'Node.js 2016 Report', eventValue: 0});">Read the full report.</a>
```

Comment:
I have a different idea here. Instead of adding this JavaScript code inline to all links, a data-attribute would be more appropriate, e.g:

```
<a href="/static/documents/2016-survey-report.pdf" data-casestudy="Node.js 2016 Report">Read the full report.</a>
```

At the end of the page, add the following snippet:
```js
<script>
if (!ga) { return; }
var caseStudies = document.querySelectorAll('a[data-casestudy]');
for (var i = 0; i < caseStudies.length; i++) {
    caseStudies[i].addEventListener('click', function(event) {
      ga('send', 'event', { eventCategory: 'casestudy', eventAction: 'click', eventLabel: this.getAttribute('data-casestudy'), eventValue: 0});
    });
}
</script>
```

This selects all links on that page that have the attribute `data-casestudy` and adds an event listener (basically the same way that you did before), which fires the Google Analytics event on click. But instead of adding the tracking code over and over for each link, it's in one place. So if you need to make adjustments for the tracking code, there is only one place to edit.